elide-async/src/main/java/com/yahoo/elide/async/service/AsyncCleanerService.java

```diff
@@ -25,7 +25,7 @@
 @Slf4j
 public class AsyncCleanerService {
 
-    private final int defaultCleanupDelayMinutes = 360;
+    private final int defaultCleanupDelayMinutes = 120;
```

Comment:
The cleaner thread is responsible for updating the status of interrupted async queries (due to app crash or restart) to TIMEDOUT from PROCESSING. It is also responsible for cleaning up history. Tweaking to lower the value.